src/views/recommended/recommended_item_view.js

```diff
@@ -20,5 +25,19 @@ export default BaseView.extend({
 
   sendSelectionDetails (event) {
     webChannel.sendUrlSelected(this.model.url, 'url');
+  },
+
+  injectPreview () {
+    if (this.model.image) {
+      this.preview = new Image('img');
+      this.preview.onload = () => { this.showLoadedPreview(); };
+      this.preview.onerror = () => {};
+      this.preview.src = this.model.image.url;
+      this.previewContainer.appendChild(this.preview);
+    }
+  },
```

Comment:
Seems like this is the same code as `ActivityItemView.injectFavicon`. Seems like we should avoid duplicating the code, any thoughts on how to do that? Might be a nice opportunity to create a base class just for these row items. I guess we could also move part of the code to some kind of a view helper utils file, but it's hard to see how to do that cleanly (maybe you have ideas).